src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -55,12 +55,12 @@ If you're brand new to New Relic and want to see how easy using NRQL can be, you
   </Collapser>
 </CollapserGroup>
 
-## Syntax [#syntax]
+## Syntax and rules [#syntax]
 
-This resource contains explanations for the various functions and clauses available for use in a NRQL query. Other popular resources for understanding NRQL include:
+This resource contains explanations for the various functions and clauses available for use in a NRQL query. Other popular resources for understanding NRQL syntax and rules include:
 
 * [Intro to NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql): provides a brief overview of NRQL as a whole, and links to other introductory docs
-* [How to use NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-how-nrql-works): explains how NRQL works and how to use it to query data
+* [How to use NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-how-nrql-works): explains some basic NRQL syntax rules, and how to use it to query your data
```

Comment:
"it" no longer has an antecedent.  Maybe "...and how to use NRQL to query your data"?